lib/sycamore/sycamore/transforms/extract_entity.py

```diff
@@ -126,6 +144,42 @@ def _handle_element_prompting(self, document: Document) -> Any:
         else:
             return self._get_entities(content)
 
+    def _handle_element_chunking(self, document: Document) -> Any:
+        assert self._tokenizer is not None
+        ind = 0
+        while ind < len(document.elements):
```

Comment:
can you combine this scrolling logic with what's in llm filter

Reply:
done